src/stdune/result.ml

```diff
@@ -59,4 +59,22 @@ let concat_map =
   in
   fun l ~f -> loop f [] l
 
+let rec iter t ~f =
```

Comment:
Shouldn't this be `Result.List.iter`?

Reply:
Yeah, I could also move a few other functions there too.